src/osd/osd_types.h

```diff
@@ -970,6 +1004,18 @@ inline std::ostream& operator<<(std::ostream& out, const eversion_t& e) {
   return out << e.epoch << "'" << e.version;
 }
 
+template <> struct fmt::formatter<eversion_t> {
+  constexpr auto parse(format_parse_context& ctx)
+  {
+    return ctx.begin();
+  }
+
+  template <typename FormatContext> auto format(eversion_t const& ev, FormatContext& ctx)
```

Comment:
could you put `auto  format(eversion_t const& ev, FormatContext& ctx)` at a new line?